meta/samples_repository/README.md

```diff
@@ -0,0 +1,102 @@
+# Best practices using the samples repository
```

Comment:
Should this README talk about the fact that this sample playbook would require `git` to be installed on the control node?